### ISSUE TITLE
JwtConfigurer should be fluent

### ISSUE THREAD

**Issue by jgrandja** [spring-projects/spring-security#5595]
The `JwtConfigurer` currently reads like this:

```java
http
  .oauth2()
    .resourceServer()
      .jwt()
        .jwkSetUri(jwkSetUri)
      .jwt()
        .decoder(decoder);
```

However, it needs to read like this:

```java
http
  .oauth2()
    .resourceServer()
      .jwt()
        .jwkSetUri(jwkSetUri)
        .decoder(decoder);
```

**Comment by jzheaux** [comment]
@jgrandja If you don't mind, I'll assign this to myself once I am working on it. That way, the community can jump in if they get to it before me.

**Comment by dnl50** [comment]
Implemented the suggested improvements in #5644
